practice/pyramid/site.css

```diff
@@ -0,0 +1,26 @@
+div {
+  border: 1px solid black;
+}
+
+.row {
+  margin: 20px;
+  display: flex;
+  height: 100px;
+  box-sizing: border-box;
+  justify-content: space-between;
+}
+
+div.inner {
+  flex: 1;
+  margin: 10px;
+}
+
+div.inner.greater{
+  flex: 4;
+}
+
+div.inner.small{
```

Comment:
Use the least specific selectors to achieve your goal. `.small` and `.greater` would be enough here.